app/controllers/application_controller.rb

```diff
@@ -0,0 +1,9 @@
+class ApplicationController < ActionController::Base
+    def hello
+        render html: "¡Hola, world!"
+    end
+
+    def goodbye
+        render html: "goodbye, world!testtest"
+    end
+end
```

Comment:
```suggestion
class ApplicationController < ActionController::Base
  def hello
    render html: "¡Hola, world!"
  end

  def goodbye
    render html: "goodbye, world!testtest"
  end
end
```

ruby ではインデントはスペース2文字分で表現するのが慣習です！
エディタの設定でインデント幅を指定したり、↓のようなコード解析ツール（linter と呼ばれるものです）を利用するのがおすすめです。
https://qiita.com/d0ne1s/items/a3566aca023d11f2bfc4